src/utils/sort.ts

```diff
@@ -27,16 +26,16 @@ export function sortByDomain(data, property, direction = 'asc', domain) {
 
 export function sortByTime(data, property, direction = 'asc') {
   return data.sort((a, b) => {
-    const aDate = moment(a[property]);
-    const bDate = moment(b[property]);
+    const aDate = a[property].getTime();
+    const bDate = b[property].getTime();
```

Comment:
That's a good point, also being explicit, even though (or perhaps because) it requires a few more lines of code is usually better for understanding what's going on anyway :)